docs/user_guides/fs/storage_connector/creation/redshift.md

```diff
@@ -38,15 +41,17 @@ Enter the details for your Redshift connector. Start by giving it a **name** and
 
 1. Select "Redshift" as connector protocol.
 2. The name of the cluster.
-3. The name of the database driver class.
-4. The database endpoint.
-5. The database name.
-6. The database port.
-7. The database username, here you have the possibility to let Hopsworks auto-create the username for you.
+3. The database endpoint. Should be in the format of `[UUID].eu-west-1.redshift.amazonaws.com`. For example, if the endpoint info 
+   displayed in Redshift is `cluster-id.uuid.eu-north-1.redshift.amazonaws.com:5439/dev` the value to enter 
+   here is just `uuid.eu-north-1.redshift.amazonaws.com` 
+4. The database name.
+5. The database port.
+6. The database username, here you have the possibility to let Hopsworks auto-create the username for you.
 8. Database Driver (optional): You can use the default JDBC Redshift Driver `com.amazon.redshift.jdbc42.Driver` 
    included in Hopsworks or set a different driver (More on this later).
-9. Optionally provide the database group and table to point the connector to.
-9. Set the appropriate authentication method.
+9. Optionally provide the database group and table to point the connector to. A database group is the group created 
```

Comment:
Don't finish a sentence on a preposition 'to'

 Optionally provide the database group and table as the target of the connector.
 or
 Optionally provide the database group and table for the connector.